### ISSUE TITLE
Who or what should we connect with at MozFest?

### ISSUE THREAD

**Issue by esylvan** [NEXMAP/odom_MozFest2017#1]
Add your ideas to this issue!

**Comment by daniellecrobinson** [comment]
@npscience - check out this project!

**Comment by esylvan** [comment]
From the mozillascience call okfn: [https://github.com/okfn/](https://github.com/okfn/)

Open Knowledge International

**Comment by esylvan** [comment]
From the mozillascience call ckan: [https://github.com/ckan//](https://github.com/ckan/)

ckan
CKAN is an open-source DMS (data management system) for powering data hubs and data portals. CKAN makes it easy to publish, share and use data. It powers datahub.io, catalog.data.gov and data.gov.uk among many other sites.

**Comment by npscience** [comment]
Yes @daniellecrobinson! @esylvan has kindly agreed to chat at MozFest - I’m looking forward to it (and may pack my electronic circuitboard paint....) :sparkles:
